offices/_single.html

```diff
@@ -179,13 +179,7 @@ <h2 class="h4 u-hide-on-mobile">
                 block__flush-sides">
         {% if activities_feed %}
         <section class="office_activities">
-            <h1 class="header-slug">
-                <span class="header-slug_inner">
-                    Latest Office Activities
-                </span>
-            </h1>
-            {{ activities_feed }}
-            <!-- link to activities page? -->
+            {% include 'templates/activities-feed.html' %}
             <a class="jump-link jump-link__right" href="/activity-log/">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/a0feb566b64bc96a4bdf992b73df7091f3a04488